ASM/c/gfx.c

```diff
@@ -214,6 +216,12 @@ void sprite_draw(z64_disp_buf_t* db, sprite_t* sprite, int tile_index,
             width_factor, height_factor);
 }
 
+void rando_display_buffer_init() {
```

Comment:
```suggestion
#define RANDO_DB_SIZE 0x4000
void rando_display_buffer_init() {
```

Reply:
Went with `const unsigned int`s near where the buffer pointer is defined at the top.